index.html

```diff
@@ -1,116 +1,182 @@
-        <li class="feature-item">
-            <h3>Быстро</h3>
-            <p>Мы делаем свою работу быстро! Два часа пролетят незаметно и Вы - счастливый обладатель стильной стрижки-минутки!</p>
-        </li>
-        <li class="feature-item">
-            <h3>Круто</h3>
-            <p>Забудьте, как Вы стриглись раньше. Мы селаем из Вас звезду футбола или кино! Во всяком случае внешне.</p>
+ <header class="header">
+    <nav class="navigation">
+        <a class="header-logo">
+            <img src="img/index-logo.svg" width="368" height="153" alt="Barbershop Borodinsky Logo">
+        </a>
+        
+        <ul class="site-navigation">
+            <li><a href="info.html">Информация</a></li>
+            <li><a href="news.html">Новости</a></li>
+            <li><a href="price.html">Прайс-лист</a></li>
+            <li><a href="contacts.html">Контакты</a></li>
+        </ul>
+   
+        <ul class="user-navigation">
+              <li class="login-link">Вход</li>
+        </ul>
+    </nav>
+
+ </header>
+<main class="container">
+
+        <h1 class="visually-hidden">Барбершоп "Бородинский"</h1>
+
+        <section class="features">
+                    <h2 class=visually-hidden>Приемущества</h2>
             
-        <li class="feature-item">
-            <h3>Дорого</h3>
-            <p>Наши мастера - профессионалы своего дела и не могут стоить дешево.К тому же, разве цена не дает определенный статус?</p>
-        </li>
-             </li>
-    </ul>
-</section>
-<section class="news">
-    <h2>Новости</h2>
-    <ul class="news-list">
-        <li class="news-item">
-            <p>Нам наконец завезли Ягермейстер! Теперь Вы сможете пропустить стаканчик во время стрижки</p>
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-        <li class="news-item">
-            <p>В нашей команде пополнение Борис "ПуджХукер228" Стригунец, обладатель множества титулов и наград пополнил наши стройные ряды</p>
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-    </ul>
-    <a class="button" href="news.html">Все новости о чоуках на чике</a>
-</section>
-<section class="gallery">
-    <h2>Фотогалерея</h2>
-    <figure class="gallery-section">
-        <a href="хз какая ссылка будет"><src="img/studio.jpg" width="286" height="164" alt="Интерьер"></src></a>
-    </figure>
-    <span class="button gallery-button gallery-button-back">Назад</span>
-    <span class="button gallery-button gallery-button-next">Рузке Вперде</span>
-</section>
-<section class="contacts">
-    <h2>Контактная информация</h2>
-    <p>Тут инфа<br>
-    ещё инфа<br>
-    инфа<br>
-    инфа<br>
-    инфа
-    </p>
+            <ul class="features-list">
+                <li class="feature-item">
+                    
+                    <h3>Быстро</h3>
+                    <p>Мы делаем свою работу быстро! Два часа пролетят незаметно и Вы - счастливый обладатель стильной стрижки-минутки!</p>
+                
+                </li>
```

Comment:
в общем, внутри тега, сверху и снизу отбивки лишние, везде такое поправить надо
можно между дочерними тегами ставить, но тут не так и нужно, тут читаемо в данной ситуации
можно добавить можно оставить, не так важно
```suggestion
                <li class="feature-item">
                    <h3>Быстро</h3>

                    <p>Мы делаем свою работу быстро! Два часа пролетят незаметно и Вы - счастливый обладатель стильной стрижки-минутки!</p>
                </li>
```

Reply:
блин тут еще криво отображает всё в комментах, пздс
ну тут понятно что 40-вая строка в коменте она одна